Cargo.toml

```diff
@@ -48,9 +48,9 @@ tempfile = "3.3.0"
 thiserror = "1.0.31"
 tinytemplate = "1.2.1"
 tokio = { version = "1.20.0", features = ["rt-multi-thread", "process", "sync"], default-features = false }
-toml = "0.5.9"
+toml = "0.5.8"
```

Comment:
Why are we downgrading `toml` here?

Reply:
embed-resource pins `=0.5.8` because 0.5.9 failed to build in their CI... though I'm not entirely sure why: https://ci.appveyor.com/project/nabijaczleweli/rust-embed-resource/builds/44058177/job/xqpfqv6ic05g99r8#L96

Reply:
Fair enough!